### DISCUSSION TITLE
Change repository visibility button is disabled

### DISCUSSION THREAD

**Discussion by Devkjk** [community/community#21968] [New to GitHub]
<p>I want to change repo public to private ,but Change repository visibility button is disabled.<br>
How to change to private repo.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/f/3f4756940b682c9c65c63c73cdb6aa1dd5e2c315.png" data-download-href="/uploads/short-url/91MVaf1LjtlpdccnuTgULg4egCh.png?dl=1" title="re" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/67408099/181067958-c6a53b50-33ee-4e6d-8ece-22e74d4c95cb.png" alt="re" data-base62-sha1="91MVaf1LjtlpdccnuTgULg4egCh" width="690" height="328" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/f/3f4756940b682c9c65c63c73cdb6aa1dd5e2c315_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">re</span><span class="informations">829×395 21.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jsoref** [comment, marked as answer]
<p>You’d have to create a new private repository. You could do that with <a href="https://github.com/new/import" class="inline-onebox">Sign in to GitHub · GitHub</a></p>
<p>I’m not quite sure what you’re trying to achieve. Odds are creating this private repository won’t actually do what you want.</p>
<p>Why do you want a repository that is part of a fork network to be private?</p>
<p>Have you checked the license of the code to ensure that you can make private changes without being required to publish them?</p>

**Comment by Devkjk** [comment]
<p>HI,I was misunderstanding that I signed in this repo’s owner account&gt;&lt;<br>
I resign in,then I was able to change repo private.</p>
